cumulus/pallets/xcmp-queue/src/migration.rs

```diff
@@ -48,6 +50,12 @@ impl<T: Config> OnRuntimeUpgrade for MigrationToV3<T> {
 			weight.saturating_accrue(T::DbWeight::get().writes(1));
 		}
 
+		if StorageVersion::get::<Pallet<T>>() == 3 {
```

Comment:
`STORAGE_VERSION` at line 30 needs to be changed to `4`.

Reply:
Done